From 6b428aa310cbffd74d8e0020e53e4c05a4195469 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 8 May 2018 11:02:02 +0800
Subject: [PATCH 3747/5242] MLK-18229-1 gpu: imx: dpu: fetchdecode: Cosmetic
 changes on fd_ops entries

commit  23212af787776fea4f29106d3a667bdb123fd336 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch contains cosmetic changes on fd_ops entries to make the
entries be more readable.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-fetchdecode.c |   32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-fetchdecode.c b/drivers/gpu/imx/dpu/dpu-fetchdecode.c
index 3ba59cb..acfface 100644
--- a/drivers/gpu/imx/dpu/dpu-fetchdecode.c
+++ b/drivers/gpu/imx/dpu/dpu-fetchdecode.c
@@ -647,22 +647,22 @@ void dpu_fd_put(struct dpu_fetchunit *fu)
 EXPORT_SYMBOL_GPL(dpu_fd_put);
 
 static const struct dpu_fetchunit_ops fd_ops = {
-	.set_burstlength = fetchunit_set_burstlength,
-	.set_baseaddress = fetchdecode_set_baseaddress,
-	.set_src_bpp = fetchdecode_set_src_bpp,
-	.set_src_stride = fetchdecode_set_src_stride,
-	.set_src_buf_dimensions = fetchdecode_set_src_buf_dimensions,
-	.set_fmt = fetchdecode_set_fmt,
-	.enable_src_buf = fetchdecode_enable_src_buf,
-	.disable_src_buf = fetchdecode_disable_src_buf,
-	.is_enabled = fetchdecode_is_enabled,
-	.set_framedimensions = fetchdecode_set_framedimensions,
-	.set_controltrigger = fetchdecode_set_controltrigger,
-	.get_stream_id = fetchunit_get_stream_id,
-	.set_stream_id = fetchunit_set_stream_id,
-	.pin_off = fetchunit_pin_off,
-	.unpin_off = fetchunit_unpin_off,
-	.is_pinned_off = fetchunit_is_pinned_off,
+	.set_burstlength	= fetchunit_set_burstlength,
+	.set_baseaddress	= fetchdecode_set_baseaddress,
+	.set_src_bpp		= fetchdecode_set_src_bpp,
+	.set_src_stride		= fetchdecode_set_src_stride,
+	.set_src_buf_dimensions	= fetchdecode_set_src_buf_dimensions,
+	.set_fmt		= fetchdecode_set_fmt,
+	.enable_src_buf		= fetchdecode_enable_src_buf,
+	.disable_src_buf	= fetchdecode_disable_src_buf,
+	.is_enabled		= fetchdecode_is_enabled,
+	.set_framedimensions	= fetchdecode_set_framedimensions,
+	.set_controltrigger	= fetchdecode_set_controltrigger,
+	.get_stream_id		= fetchunit_get_stream_id,
+	.set_stream_id		= fetchunit_set_stream_id,
+	.pin_off		= fetchunit_pin_off,
+	.unpin_off		= fetchunit_unpin_off,
+	.is_pinned_off		= fetchunit_is_pinned_off,
 };
 
 void _dpu_fd_init(struct dpu_soc *dpu, unsigned int id)
-- 
1.7.9.5

